tests/classes/existing_repr_test.py

```diff
@@ -0,0 +1,15 @@
+"""Test class for kw only arguments."""
```

Comment:
- Can you add another test class here without a `__repr__` to make sure that this still works.
- Add a dataclass  (which has a dynamically created `__repr__`) to make sure that it will be skipped as well

Reply:
Yup, I can add that